docs/guides/4-custom-rulesets.md

```diff
@@ -351,10 +356,12 @@ aliases:
     targets:
```

Comment:
I can't quite tell in the diff here, but is there an extra space (targets is indented more than description)?

Reply:
it looks good, verified it manually now haha
We should probably use JSON as default and have some automated tool that generates YAML examples based on it.
This way we would have 2 examples at all times, which would be actually useful.